### DISCUSSION TITLE
DC voltage as input for the the VdMot_Controller - instead of thermostat or data

### DISCUSSION THREAD

**Discussion by WA66** [Lenti84/VdMot_Controller#120] [General]
Hello,

is it possible to control several VDMOT or Equiva Model N with the VdMot_Controller via DC voltage?
Not via Thermostat, Data etc., but e. g. a control voltage input of x V on Port 1 results in adjusting the valve V1 to x %?

I want to fit a heating distribution with an electrically switchable hydraulic adjustment.
E. g. using two sets of potis to adjust every single valve to x % and then switch between the two sets of potis.

Best regards

**Comment by Lenti84** [comment]
Hi WA66,

there is no such way in controlling the Vdmot Controller.
If it is super crucial for you, one idea come to my mind: use a ESP8266 oder ESP32 as MQTT broker, give it an digital or even analog input, then send (predefined) target values for all 12 valves.

But I doubt, that this is a good idea. Valve position to flow conversion is unlikely as exact as you will need it.
Small deviations will always be compensated in a setup with control loop for the room temperature.

Best regards

**Reply by WA66** [reply]
Hi Lenti84,

thank you very much for the hints and your valuation.

Another idea is to look for some kind of precise servo motors, adapted to the mechanics.
Or to use separate flow control devices, inserted in the pipes.

Best regards
